test/benchmark.js

```diff
@@ -0,0 +1,110 @@
+
+var fs = require('fs');
+var path = require('path');
+var child_process = require('child_process')
+
+var Benchmark = require('benchmark');
+var colors = require('colors/safe');
+
+var formatNumber = Benchmark.formatNumber;
+var suite = new Benchmark.Suite();
+
+var Parser = require('../lib/nearley.js').Parser;
+var shared = require('./_shared.js');
+var compile = shared.compile
+  , read = shared.read;
+
+
+// Define benchmarks
+
+function addTest(parserName, parser, exampleInputs) {
+    exampleInputs.forEach(function(inputPath) {
+        var input;
+        var exampleName;
+        if (/^test\//.test(inputPath)) {
```

Comment:
Can't you just do `read("test1.json")` in the test specification? This seems like a hack that will cause trouble in the future.